packages/synapse-react-client/src/components/ImageCardGridWithLinks/ImageCardGridWithLinks.tsx

```diff
@@ -0,0 +1,216 @@
+      }
+    }
+    getData()
+  }, [entityId, accessToken, queryResultBundle])
+
+  const dataRows = queryResultBundle?.queryResult!.queryResults.rows ?? []
+  const linkColumnIndex = getFieldIndex(ExpectedColumns.LINK, queryResultBundle)
+  const linkTextColumnIndex = getFieldIndex(
+    ExpectedColumns.LINKTEXT,
+    queryResultBundle,
+  )
+
+  const ImageCard = ({ card, index }: ImageCardProps) => (
+    <Grid
+      item
+      xs={12}
+      sm={6}
+      md={4}
+      key={card.rowId}
+      sx={{ height: '245px', paddingTop: '24px', paddingLeft: '24px' }}
+    >
+      {isLoading ? (
+        <Skeleton variant="rectangular" height={221} width="100%" />
+      ) : (
+        <>
+          <Link
+            component={RouterLink}
+            to={card.values[linkColumnIndex] || ''}
+            sx={{
+              display: 'flex',
+              alignItems: 'center',
+              gap: '10px',
+              position: 'absolute',
+              backgroundColor: '#FFFF',
+              borderRadius: '5px 0 6px 0',
+              textDecoration: 'none',
+              '&:hover': {
+                textDecoration: 'none',
+              },
+              padding: '6px 10px',
+            }}
+          >
+            <Typography
+              color="grey.1000"
+              variant="headline2"
+              sx={{ fontSize: '16px' }}
+            >
+              {card.values[linkTextColumnIndex]}
+            </Typography>
+            <ArrowForwardIosIcon
+              style={{
+                color: 'unset',
+                width: 16,
+                height: 16,
+              }}
+            />
+          </Link>
+          <CardMedia
+            component="img"
+            image={images?.[index]}
+            style={{
+              height: '100%',
+              width: '100%',
+              borderRadius: '6px',
+              objectFit: 'cover',
+            }}
+          />
+        </>
+      )}
+    </Grid>
+  )
+
+  return (
+    <Box
+      sx={{
+        display: 'grid',
+        gridTemplateColumns: { xs: '1fr', md: '1fr 3fr' },
+        gap: { xs: '38px', md: '80px' },
+      }}
+    >
+      <Box
+        sx={{
+          padding: {
+            xs: '0 16px',
+            md: 0,
+          },
+        }}
+      >
+        <Box
+          display="flex"
+          flexDirection="column"
+          gap="16px"
+          sx={{ borderTop: '3px solid', borderColor: 'grey.400' }}
+        >
+          <Typography color="grey.1000" variant="headline2" paddingTop="26px">
+            {title}
+          </Typography>
+          <Typography
+            variant="body1"
+            color="grey.800"
+            sx={{ fontStyle: 'italic' }}
+          >
+            {summaryText}
+          </Typography>
+        </Box>
+      </Box>
+      <Grid
+        container
+        spacing={2}
+        sx={{
+          order: { xs: 1, md: 0 },
+        }}
+      >
+        <Grid container spacing={2.5} sx={{ margin: 0 }}>
```

Comment:
Is this double-grid wrapper necessary? It's ok if it is, I'm just having trouble figuring out what the purpose is

Reply:
Must've not cleaned it up right when I was refactoring! Removed one of the Grid wrappers as it's unnecessary.